x-pack/test/api_integration/deployment_agnostic/apis/observability/slo/create_slo.ts

```diff
@@ -0,0 +1,241 @@
+            timestampField: '@timestamp',
+            total: 'container.cpu.user.pct: *',
+          },
+          type: 'sli.kql.custom',
+        },
+        name: 'Test SLO for api integration',
+        objective: {
+          target: 0.99,
+        },
+        revision: 1,
+        settings: {
+          frequency: '1m',
+          syncDelay: '1m',
+          preventInitialBackfill: false,
+        },
+        tags: ['test'],
+        timeWindow: {
+          duration: '7d',
+          type: 'rolling',
+        },
+        version: 2,
+      });
+
+      const rollUpTransformResponse = await transformHelper.assertExist(getSLOTransformId(id, 1));
+      expect(rollUpTransformResponse.transforms[0].source.index).eql(['kbn-data-forge*']);
+      expect(rollUpTransformResponse.transforms[0].dest).eql({
+        index: '.slo-observability.sli-v3.3',
+        pipeline: `.slo-observability.sli.pipeline-${id}-1`,
+      });
+      expect(rollUpTransformResponse.transforms[0].pivot.group_by).eql({
+        'slo.groupings.tags': { terms: { field: 'tags' } },
+        '@timestamp': { date_histogram: { field: '@timestamp', fixed_interval: '1m' } },
+      });
+
+      const summaryTransformResponse = await transformHelper.assertExist(
+        getSLOSummaryTransformId(id, 1)
+      );
+      expect(summaryTransformResponse.transforms[0].source.index).eql([
+        '.slo-observability.sli-v3.3*',
+      ]);
+      expect(summaryTransformResponse.transforms[0].dest).eql({
+        index: '.slo-observability.summary-v3.3',
+        pipeline: `.slo-observability.summary.pipeline-${id}-1`,
+      });
+    });
+
+    describe('groupBy smoke tests', () => {
+      it('creates instanceId for SLOs with multi groupBy', async () => {
+        const apiResponse = await sloApi.create(
+          Object.assign({}, DEFAULT_SLO, { groupBy: ['system.network.name', 'event.dataset'] }),
+          adminRoleAuthc
+        );
+
+        expect(apiResponse).property('id');
+        const { id } = apiResponse;
+
+        await retry.tryForTime(300 * 1000, async () => {
+          const response = await esClient.search(getRollupDataEsQuery(id));
+
+          // @ts-ignore
+          expect(response.aggregations?.last_doc.hits?.hits[0]._source.slo.instanceId).eql(
+            'eth1,system.network'
+          );
+        });
+      });
+
+      it('creates instanceId for SLOs with single groupBy', async () => {
+        const apiResponse = await sloApi.create(
+          Object.assign({}, DEFAULT_SLO, { groupBy: 'system.network.name' }),
+          adminRoleAuthc
+        );
+
+        expect(apiResponse).property('id');
+        const { id } = apiResponse;
+
+        await retry.tryForTime(300 * 1000, async () => {
+          const response = await esClient.search(getRollupDataEsQuery(id));
+
+          // @ts-ignore
+          expect(response.aggregations?.last_doc.hits?.hits[0]._source.slo.instanceId).eql('eth1');
+        });
+      });
```

Comment:
I didn't notice it in first place, but if retry call is taking 5 min, there is a chance of a test suite to timeout (Mocha suite timeout is 6 min)
It is pretty annoying failure, because you won't get an error message about what caused the timeout.

I suggest checking few options:
- decrease retry timeout (3 min is a max we usually suggest to put)
- increase suite timeout if test run might be slow and no way to speed it up

Reply:
I think 3min should be enough, on serverless locally it takes like 10s